clippy.toml

```diff
@@ -0,0 +1 @@
+allow-expect-in-tests = true
```

Comment:
Do we need the `#[allow::clippy(unwrap_used)]` now that this is set?

Reply:
Oh likely not now that we've got this set! I have some more commits to push, I'll include this